### DISCUSSION TITLE
Mobile (Andriod/IOS) - Auto open keyboard when pressing with mouse on typing area

### DISCUSSION THREAD

**Discussion by yairglk** [rustdesk/rustdesk#9943] [Feature Request]
As far many tests, only TeamViewer does this correctly, I hope you guys can do it too:

What does that mean? When a user is connecting to his PC from his mobile - in places where you need to type and you will press with the mouse - the keyboard should automatically open/popup/appear.

Current situation - You need to press on the keyboard Icon to open the keyboard,  this is time-consuming, very uncomfortable

You will find it weird, but the most important feature is easily accessing their computer from their phones (Android/Apple) comfortably.

**Comment by fufesou** [comment]
It's a good suggestion. But We have some other tasks now.
We'll may implement later.

**Reply by yairglk** [reply]
Kind of reply that tells you that it will be never be done because there are more important things.
so my question is, is there a way to pay for a feature like this? its important
